frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/LinkedReleasesAndProjectsAwarePortlet.java

```diff
@@ -37,9 +36,7 @@
 import java.util.stream.Collectors;
 
 import static org.eclipse.sw360.datahandler.common.CommonUtils.nullToEmptyList;
-import static org.eclipse.sw360.portal.common.PortalConstants.PARENT_BRANCH_ID;
-import static org.eclipse.sw360.portal.common.PortalConstants.PROJECT_LIST;
-import static org.eclipse.sw360.portal.common.PortalConstants.RELEASE_LIST;
+import static org.eclipse.sw360.portal.common.PortalConstants.*;
```

Comment:
This doesn't look like a necessary change. Maybe even the opposite of that. I think you can revert this file

Reply:
yepp, reverted all changes regarding this file.